src/eradiate/scenes/phase/_blend.py

```diff
@@ -15,6 +16,23 @@
 from ...units import unit_context_kernel as uck
 
 
+def _convert_weights(value: npt.ArrayLike) -> npt.NDArray[np.float64]:
```

Comment:
For consistency with the `attrs` decorator naming (rule used elsewhere in the code base; the rest is a consequence of my comment on Numpy typing):
```suggestion
def _weights_converter(value: np.typing.ArrayLike) -> np.ndarray:
```